google/provider_test.go

```diff
@@ -206,3 +212,100 @@ func multiEnvSearch(ks []string) string {
 	}
 	return ""
 }
+
+/**
+* bootstrapKMSkey will return a KMS key that can be used in tests that are
+* testing KMS integration with other resources.
+*
+* This will either return an existing key or create one if it hasn't been created
+* in the project yet. The motivation is because keyrings don't get deleted and we
+* don't want a linear growth of disabled keyrings in a project but we also don't want
+* to incur the overhead of creating a new project for each test that needs to use
+* a KMS key.
+**/
+var SharedKeyRing = "tftest-shared-keyring-1"
```

Comment:
Why "-1"?

Reply:
It's for versioning. I added it for my own testing to make sure that new keys were getting created correctly when the constant changed. I left it because if we need to change it in future we can version bump to 2. I could check if the key regex support a 'v1' syntax.